lib/tasks/bulk.rake

```diff
@@ -11,4 +11,80 @@ namespace :bulk do
       DeliveryRequestWorker.perform_async_in_queue(id, queue: :delivery_immediate)
     end
   end
+
+  desc "Switch immediate subscribers of the specified list slugs to daily digest"
+  task switch_to_daily_digest: :environment do |_t, args|
+    lists = SubscriberList.where(slug: args.extras)
+    raise "One or more lists were not found" if lists.count != args.extras.count
+
+    subscriptions = Subscription.active.immediately.where(subscriber_list: lists)
+    raise "No subscriptions to change" if subscriptions.none?
+
+    subscribers = Subscriber.where(id: subscriptions.pluck(:subscriber_id)).index_by(&:id)
```

Comment:
Do you know what size the `subscriptions.pluck(:subscriber_id)` array might reach? I have a recollection of us hitting an issue on this project when we exceeded 2^16 (~65,000) items in an `IN` clause, but can't find the project. Googling for it seems to conflict between things [saying no limit](https://stackoverflow.com/questions/1009706/postgresql-max-number-of-parameters-in-in-clause#:~:text=There%20is%20no%20limit%20to,in%20the%20array%20or%20not.) and [others saying ~32,000](https://github.com/jOOQ/jOOQ/issues/5701).

Reply:
Hmm...tricky, I see what you mean. If this is an issue, it should come up when testing on Integration (TODO).

Reply:
Looks like it works 🎉